src/hotspot/share/opto/compile.hpp

```diff
@@ -53,6 +53,7 @@ class Block;
 class Bundle;
 class CallGenerator;
 class CallStaticJavaNode;
+class CastIINode;
```

Comment:
Is this required? There is no additional change/use of `CastIINode` in this file.